sharepoint/sharepoint-ps/sharepoint-online/Set-SPOListVersionPolicy.md

```diff
@@ -0,0 +1,178 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-spolistversionpolicy
+applicable: SharePoint Online
+title: Set-SPOListVersionPolicy
+schema: 2.0.0
+author: msjennywu
+ms.author: jennywu
+ms.reviewer:
+manager: seanmc
+---
+
+# Set-SPOListVersionPolicy
+
+## SYNOPSIS
+
+> [!NOTE]
+> This feature is part of the version history controls preview. If your tenant is not part of the preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
+
+Sets the version policy setting on the cocument library.
+
+## SYNTAX
+
+```powershell
+Set-SPOListVersionPolicy [-Site] <SpoSitePipeBind> [-List] <SpoListPipeBind> [-EnableAutoExpirationVersionTrim <Boolean>] [-MajorVersionLimit <int>] [-MajorWithMinorVersionsLimit <int>] [-ExpireVersionsAfterDays <int>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Sets the version policy setting on the document library.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+Set-SPOListVersionPolicy -Site https://contoso.sharepoint.com/sites/site1 -List "Documents" -EnableAutoExpirationVersionTrim $true
+```
+Example 1 sets Automatic version Storage Limits on the document library called "Documents".
+
+### EXAMPLE 2
+
+```powershell
+Set-SPOListVersionPolicy -Site https://contoso.sharepoint.com/sites/site1 -List "Documents" -EnableAutoExpirationVersionTrim $false -MajorVersionLimit 500 -MajorWithMinorVersionsLimit 20 -ExpireVersionsAfterDays 30
+```
+
+Example 2 sets Manual Version Storage Limits on the document library called "Documents" by limiting the number of versions and the time (in days) versions are kept.
+
+### EXAMPLE 3
+
+```powershell
+Set-SPOListVersionPolicy -Site https://contoso.sharepoint.com/sites/site1 -List "Documents" -EnableAutoExpirationVersionTrim $false -MajorVersionLimit 500 -MajorWithMinorVersionsLimit 20 -ExpireVersionsAfterDays 0
+```
+
+Example 3 sets Manual Version Storage Limits on the document library called "Documents" by limiting the number of versions with no time limits. The new document libraries will use this version setting.
+
+## PARAMETERS
+
+### -Site
+
+Specifies the URL of the site.
+
+```yaml
+Type: SpoSitePipeBind
+Parameter Sets: (All)
+Aliases:
+Applicable: SharePoint Online
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -List
+
+The document library name or Id.
+
+```yaml
+Type: SPOListPipeBind
+Parameter Sets: (All)
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EnableAutoExpirationVersionTrim
+Global and SharePoint Administrators in Microsoft 365 can set document library level version history limits settings that universally apply to new versions created.
```

Comment:
> in Microsoft 365 [](http://example.com/codeflow?start=36&length=17)

can remove " in Microsoft 365" as that's well understood

Reply:
Removed it from other places in set-spotenant and set-spotenant